services/blockchain-indexer/shared/indexer/accountIndex.js

```diff
@@ -39,10 +47,17 @@ const getAccountsTable = () => getTableInstance(accountsTableSchema, MYSQL_ENDPO
 const updateAccountInfoPk = async (job) => {
 	const publicKey = job.data;
 
-	const account = await getAccountsByPublicKey2([publicKey]);
-	if (account.length) {
+	try {
+		const account = {
+			address: getLisk32AddressFromPublicKeyHelper(Buffer.from(publicKey, 'hex')),
+			publicKey,
+		};
+
 		const accountsTable = await getAccountsTable();
 		await accountsTable.upsert(account);
+	} catch (err) {
+		logger.warn('Failed to update accounts table. Will Retry.');
+		redis.sadd('pendingAccountsByPublicKey', publicKey);
```

Comment:
Can we please use constants for the queue names?
```suggestion
		await redis.sadd('pendingAccountsByPublicKey', publicKey);
```